src/components/Tabs/components/TabMeasurer/tests/TabMeasurer.test.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import {mountWithAppProvider} from 'test-utilities';
+import {noop} from '@shopify/javascript-utilities/other';
+import TabMeasurer from '../TabMeasurer';
+import Tab from '../../Tab';
+import Item from '../../Item';
+
+describe('<TabMeasurer />', () => {
+  const mockProps = {
+    tabToFocus: 0,
+    activator: <Item id="id" focused />,
+    selected: 1,
+    tabs: [
+      {
+        id: 'repeat-customers',
+        content: 'repeat-customers',
+      },
+      {
+        id: 'prospects',
+        content: 'prospects',
+      },
+    ],
+    siblingTabHasFocus: false,
+    handleMeasurement: noop,
+  };
+
+  describe('tabToFocus', () => {
+    it('is used to determine correct focused value of Tab', () => {
```

Comment:
Same here as above you probably want this to be two tests. Usually this is the case as you usually don't wanna do multiple assertions in a single test. 😄